packages/core/src/language-server/glint-language-server.ts

```diff
@@ -416,6 +416,45 @@ export default class GlintLanguageServer {
     return this.glintConfig.environment.isTypedScript(file) ? 'typescript' : 'javascript';
   }
 
+  public organizeImports(
+    uri: string,
+    formatOptions: ts.FormatCodeSettings = {},
+    preferences: ts.UserPreferences = {}
+  ): TextEdit[] {
+    const transformInfo = this.transformManager.findTransformInfoForOriginalFile(
+      uriToFilePath(uri)
+    );
+
+    if (!transformInfo) {
```

Comment:
An `organizeImports` call should still be actionable even if we haven't done any transformation on the file, as long as it's something we recognize as one of our known script extensions, right?

Reply:
Not sure I completely understand, but I only had to get the transformInfo in order for `.gts` files. Passing them to tsserver directly would error. Using the original file name seemed to work fine for `.ts` files.

Reply:
As this stands, invoking this command on a regular `.ts` file with no accompanying template won't do anything, since we're short-circuiting here. Unless I'm misreading the code?

Reply:
`transformInfo` will only be `null` for `.hbs` files without a corresponding `.ts`.  For regular TS files, a `transformInfo` object is returned with a valid `transformedFileName` but a null `transformedModule`.

Since this command is not available in `.hbs` files, I don't think it's a major concern

Reply:
> For regular TS files, a transformInfo object is returned with a valid transformedFileName but a null transformedModule.

Ahh, that's what I get for trying to review from memory on code I haven't actually looked at in detail for a long time. Thanks for clarifying!